ibm/provider_test.go

```diff
@@ -537,6 +542,22 @@ func testAccPreCheck(t *testing.T) {
 	}
 }
 
+func testAccPreCheckEnterprise(t *testing.T) {
+	if v := os.Getenv("IC_API_KEY"); v == "" {
+		t.Fatal("IC_API_KEY must be set for acceptance tests")
+	}
+
+}
+
+func testAccPreCheckEnterpriseAccountImport(t *testing.T) {
```

Comment:
@hkantare So, this test case is special because we need ACCOUNT_TO_BE_IMPORTED to be present for running this test case.
Basically, this is test case where a stand-alone account, which was not part of enterprise to be imported into the enterprise.